### ISSUE TITLE
Panning or zooming out causes raw image to disappear

### ISSUE THREAD

**Issue by xies** [ilastik/ilastik#1634]
I am running OSX El Capitan (10.11.6) and ilastik 1.2.2

When analyzing a large TXY image (20x2000x2000), the raw data channel initially is centered at 100% zoom on the image, as expected:
<img width="500" alt="screen shot 2017-10-27 at 17 46 10 pm" src="https://user-images.githubusercontent.com/5126258/32129647-d1ea2f04-bb3e-11e7-9eaf-1e56a6a4318b.png">

But as soon as I pan to another region or zoom out to a "fit entire screen" view, the channel completely disappears into a black screen. (The segmented objects shows that this still the same image as before, if you look at the center nuclei.) Zooming back in or re-centering does not fix this.
<img width="500" alt="screen shot 2017-10-27 at 17 46 25 pm" src="https://user-images.githubusercontent.com/5126258/32129643-ce3c0e7c-bb3e-11e7-835f-363c6b50a1d8.png">

If I turn down the alpha on the raw data channel, the black image also disappears.
<img width="500" alt="screen shot 2017-10-27 at 17 49 35 pm" src="https://user-images.githubusercontent.com/5126258/32129676-367b8328-bb3f-11e7-801e-8faf791a749f.png">

**Comment by chaubold** [comment]
@xies: This is indeed weird. How much RAM does the machine have on which you saw this error? And What datatype has your raw data (e.g. `uint16`)?

@stuarteberg, @akreshuk, @k-dominik : what do we do if the image (here I actually suspect the probabilities to cause trouble as those are `float32`) does not fit into RAM?

**Comment by xies** [comment]
@chaubold I have 16GB ram and the problematic image is indeed `uint16`. This happens in both single TIF images as well as stacked TIF timeseries. In the case of the TIF stacked images, scrolling through time while initially centered on the visible image does not affect the display, but panning and zooming still does.

**Comment by xies** [comment]
@chaubold Hi! We figured out that when you pan, the image display thresholds are being reset from their original values, and these images have very shallow signal compared to the full bit-depth so it looked effectively like they were disappearing.

**Comment by chaubold** [comment]
Good catch! I am not able to reproduce this behaviour even with images of size `4000x4000`, though...
